homeassistant/helpers/template.py

```diff
@@ -2356,6 +2393,13 @@ def __init__(
         self.filters["unpack"] = struct_unpack
         self.filters["ord"] = ord
         self.filters["is_number"] = is_number
+        self.filters["is_list"] = is_list
+        self.filters["is_set"] = is_set
+        self.filters["is_tuple"] = is_tuple
+        self.filters["set"] = _set
+        self.filters["tuple"] = _tuple
+        self.filters["is_datetime"] = is_datetime
+        self.filters["is_non_string_iterable"] = is_non_string_iterable
```

Comment:
The name is somewhat misleading since it returns False for other iterables than strings.
Should we name this `is_iterable` and just document that it returns `False` if the object is str, bytes or bytearray?

Reply:
I see your point, however I don't think the `is_iterable` name is useful either. Is there a name for this class of types? `bytes` and `bytearray` aren't strings but they are string like

Reply:
per the suggestion below I changed it to `string_like`